dags/hra.py

```diff
@@ -16,6 +17,24 @@
     schedule_interval='@monthly'
 )
 
+
+def _check_hra_api_availability():
+    now = datetime.datetime.now()
+
+    is_mon_wed_or_thrusday = (now.weekday() in [0, 2, 3])
+    is_after_6am = (now.time() >= datetime.time(6, 0))
+    is_before_8am = (now.time() < datetime.time(8, 0))
+    is_between_6am_and_8am = (is_after_6am and is_before_8am)
+
+    return is_mon_wed_or_thrusday and is_between_6am_and_8am
+
+
+wait_for_hra_api_availability_sensor = PythonSensor(
+    task_id='wait_for_hra_api_availability',
+    dag=dag,
+    python_callable=_check_hra_api_availability
```

Comment:
Yes. By default, it runs every minute. As the call is very lightweight, I think we're fine with the default. This would give us more time to actually run the collector and re-run it if something fails.